cli/cli_mesh_peers.go

```diff
@@ -166,8 +166,12 @@ func selfToOutputString(peer *pb.Peer) string {
 		{Key: "OS", Value: peer.Os},
 		{Key: "Distribution", Value: peer.Distro},
 	}
+	hostname := peer.Hostname
+	if peer.Nickname != "" {
+		hostname = peer.Nickname
```

Comment:
My peer's original name is not shown? I guess it is preserved by API and it can be used to access my peer, so, it should be shown somewhere. Or is it not so?

Reply:
I'll change this to display in this case both. 
But maybe put the nickname as `titledKeyvalListToColoredString` and hostname lower? Because it seams that other apps are treating the nickname as the main "hostname" and the Nord secondary.